### PR TITLE
redo "Make Lanth Line Better (mainly Samarium)"

### PR DESCRIPTION
**Pull request by Nxer**

### CODE REVIEW

File: src/main/java/com/elisis/gtnhlanth/loader/RecipeLoader.java [hunk 1]
```diff
@@ -987,11 +987,11 @@ public static void loadLanthanideRecipes() {
         // Digester to produce Lanthanum Chloride Concentrate
         GT_Values.RA.stdBuilder().itemInputs(GT_OreDictUnificator.get(OrePrefixes.crushed, Materials.Lanthanum, 1))
                 .itemOutputs(Materials.SiliconDioxide.getDust(3)).fluidInputs(Materials.Chlorine.getGas(36000))
-                .fluidOutputs(LanthanumChlorideConcentrate.getFluidOrGas(3000)).specialValue(800).eut(114514)
+                .fluidOutputs(LanthanumChlorideConcentrate.getFluidOrGas(3000)).specialValue(800).eut(122880)
```
Comment (boubou19) [inline src/main/java/com/elisis/gtnhlanth/loader/RecipeLoader.java]: use TierEU.RECIPE_ZPM

File: src/main/java/com/elisis/gtnhlanth/loader/RecipeLoader.java [hunk 2]
```diff
@@ -987,11 +987,11 @@ public static void loadLanthanideRecipes() {
...
                 .duration(40).addTo(RecipeAdder.instance.DigesterRecipes);
```
Comment (boubou19) [inline src/main/java/com/elisis/gtnhlanth/loader/RecipeLoader.java]: decompose the tick duration in a unique h/min/s/t way, using the TICKS, SECONDS, MINUTES, HOURS constants, for instance 40 is 2*SECONDS

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by boubou19** [comment]
Also i feel like this PR is not properly done, previous one had a 2k changes, now this one only has 58.

**Review by boubou19 — APPROVED** [review]

**Comment by Dream-Master** [comment]
@Nxer can you resolve the conflict?

**Comment by Nxer** [comment]
> @Nxer can you resolve the conflict?

where
I'll check again , now way home.

**Review by boubou19 — APPROVED** [review]

**Pull request merged**